tests/testsuite/main.rs

```diff
@@ -1,4 +1,6 @@
-// See src/cargo/lib.rs for notes on these lint settings.
+//! See src/cargo/lib.rs for notes on these lint settings.
+
+// TODO: consider removing these lint attributes when `-Zlints` hits stable.
 #![warn(rust_2018_idioms)]
 #![allow(clippy::all)]
```

Comment:
[attributes have precedence](https://github.com/rust-lang/cargo/blob/master/tests/testsuite/lints.rs#L385) which reduces how much we can really dogfood this feature...

Reply:
That's sadly true 🥲

however, apart from `cargo` we still have a bunch of crates out of control of linters. We keep them as-is so they can benefit from this dogfooding.